substreams/Cargo.toml

```
@@ -24,6 +24,9 @@ prost = "0.11"
 prost-types = "0.11"
 substreams-macro = { workspace = true }
 thiserror = "1"
+pest= "2.7.10"
+pest_derive = "2.7.10"
+rstest = "0.19.0" 
```

Pull from the workspace dependencies set
```suggestion
pest = { workspace = true }
pest_derive = { workspace = true }
rstest = { workspace = true } 
```